compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -151,7 +152,18 @@ fn rustc_terminator_to_terminator(
         Return => Terminator::Return,
         Unreachable => Terminator::Unreachable,
         Drop { .. } => todo!(),
-        Call { .. } => todo!(),
+        Call { func, args, destination, target, unwind, from_hir_call: _, fn_span: _ } => {
+            Terminator::Call {
+                func: rustc_op_to_op(func),
+                args: args.iter().map(|arg| rustc_op_to_op(arg)).collect(),
+                destination: rustc_place_to_place(destination),
+                target: target.map(|t| t.as_usize()),
+                cleanup: match unwind {
+                    UnwindAction::Cleanup(bb) => Some(bb.as_usize()),
+                    _ => None,
+                },
```

Comment:
We should probably copy the `UnwindAction` enum to smir instead of losing the information about which behaviour is desired here.

Reply:
Agreed, made the change but only applied to `Call`. We would need to apply to `Assert`, etc but we can do that as we go I guess.